docs/DeveloperGuide.adoc

```diff
@@ -1486,16 +1551,72 @@ These instructions only provide a starting point for testers to work on; testers
 
 ==== Leave application
 
+. Requesting leave application
+
+.. Prerequisites: User must be logged in.
+.. Test case: `leave date/01/01/2019` +
+   Expected: Leave application requested. Details of the leave application will be reflected in the leave panel.
+.. Test case: `leave date/01/02/2018` +
+   Expected: No leave application created. Error details shown in the status message.
+
 ==== List leave application
 
+. Listing leave applications
```

Comment:
For listing leave Instructions for Manual Testing can be removed since not really needed

Reply:
I think its better to put there. Thanks

Reply:
From our side we all have removed already, I don't think there is a point to show listing leave on the Instructions for Manual testing. Thanks

Reply:
Noted.